speedscale/README.md

```diff
@@ -0,0 +1,54 @@
+# Agent Check: Speedscale
+
+## Overview
+
+This integration publishes the results from [Speedscale][1] into Datadog.
```

Comment:
Would you be able to add a bit more detail here about the kind of data that Speedscale collects, and what types of data it's sending into Datadog?

Reply:
Thank you, that section was definitely too vague, I added more details.